api/v1alpha1/release_types.go

```diff
@@ -130,6 +130,9 @@ type ProcessingInfo struct {
 
 // ValidationInfo defines the observed state of the release validation.
 type ValidationInfo struct {
+	// Automated indicates whether the Release was created by the integration service
+	Automated bool `json:"automated,omitempty"`
```

Comment:
How is this related to validation info? It shouldn't be here. Instead, this should probably be a top level field.

Reply:
Ack